spec/buildkite/builder/github_spec.rb

```diff
@@ -78,4 +79,43 @@
       end
     end
   end
+
+  describe '#pull_request' do
+    subject { github.pull_request }
+    let(:init_uri) { URI.join(described_class::BASE_URI, "repos/Gusto/buildkite-builder/pulls/12345") }
+    let(:response_1) do
+      instance_double(Net::HTTPOK, body: JSON.dump(pull_request_response))
+    end
+    let(:pull_request_response) do 
```

Comment:
This test doesn't seem particularly useful, besides testing that the correct endpoint is hit. Any recommendations on improving?